docs/UserGuide.md

```diff
@@ -33,10 +33,10 @@ SellSavvy is **optimized for users who prefer Command Line Interface** (CLI) whi
 
    * `addcustomer n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a customer named `John Doe` to the displayed customer list.
 
-   * `deletecustomer 3` : Deletes the 3rd customer contact shown in the displayed customer list.
+   * `deletecustomer 3` : Deletes the 3rd customer's contact and order list shown in the displayed customer list and the customer's order list.
```

Comment:
order list is not shown is displayed customer list . I also not sure how to fix.
@kaixuan477 help please :(

Reply:
```suggestion
   * `deletecustomer 3` : Deletes the 3rd customer's contact shown in the displayed customer list and their corresponding order list.
```